docs/getting-started/detections-req.asciidoc

```diff
@@ -74,24 +73,32 @@ image::images/sec-admin-user.png[]
 [[access-detections-ui]]
 == Access and use Detections
 
-After enabling Detections, only users with these permission can view and use the
-*Detections* page:
+After enabling Detections, only users with these permissions can view and use rules and alerts on the *Detections* page:
 
-* {kib} space `All` privileges for the `Security`  and `Saved Objects
-Management` features
+**All**
+
+These permissions are required for both rule and alert management:
+
+* {kib} space with `All` privileges enabled for `Security`.
+* The `maintenance` permission for `.siem-signals-<kib-space>`.
 * The `read`, `write`, and `view_index_metadata` index privileges for all of these system indices:
```

Comment:
```suggestion
* The `read`, `write`, `view_index_metadata`, and `maintenance` index privileges for all of these system indices:
```

Reply:
This should match the image, where all three indices get the described privileges.